staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -550,7 +553,7 @@ type CronJobSpec struct {
 	FailedJobsHistoryLimit *int32 `json:"failedJobsHistoryLimit,omitempty" protobuf:"varint,7,opt,name=failedJobsHistoryLimit"`
 }
 
-// ConcurrencyPolicy describes how the job will be handled.
```

Comment:
revert

Reply:
concurencyPolicy is mentinoed [here](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/cron-job-v1/#CronJobSpec) and seemed like a field of CronSpec. Should we still use capital first letter?